csharp/src/Drivers/Apache/Spark/SparkHttpConnection.cs

```diff
@@ -135,11 +135,17 @@ protected override void ValidateOptions()
                     ? connectTimeoutMsValue
                     : throw new ArgumentOutOfRangeException(SparkParameters.ConnectTimeoutMilliseconds, connectTimeoutMs, $"must be a value of 0 (infinite) or between 1 .. {int.MaxValue}. default is 30000 milliseconds.");
             }
+
             TlsOptions = HiveServer2TlsImpl.GetHttpTlsOptions(Properties);
         }
 
         internal override IArrowArrayStream NewReader<T>(T statement, Schema schema, TGetResultSetMetadataResp? metadataResp = null) => new HiveServer2Reader(statement, schema, dataTypeConversion: statement.Connection.DataTypeConversion);
 
+        protected virtual HttpMessageHandler CreateHttpHandler()

```

Comment:
The linter is complaining about something whitespace-related here, but I don't see what it could be. Consider using autoformatting as per the other comment, and perhaps there's some mix of LF and CRLF line endings?